src/db/roles.rs

```diff
@@ -86,9 +86,4 @@ impl Role {
     pub fn to_bson(&self) -> Bson {
         self.clone().into()
     }
-
```

Comment:
Given that clippy is unhappy about the name of this metthod and that it's deprecated, I removed it since there are already breaking changes in this PR.